### DISCUSSION TITLE
Update save() error

### DISCUSSION THREAD

**Discussion by mis-uep** [phalcon/cphalcon#15589] [Q&A] [answered]
I have trouble saving update with this error. Please help! Thanks!

Fatal error: Uncaught Error: Call to undefined method Phalcon\Mvc\Model\Resultset\Simple::save()

Here is my code:
$updateOerl = Oerl::find("StudID = '$resultStud->StudID' AND SysNum = $result->SysOsbj");
$updateOerl->SysNum = $result3->SysNum;
$updateOerl->save();

**Comment by quasipickle** [comment, marked as answer]
`find()` returns a result set, not a single record.  What you want to use is `findFirst()`.  You also want to use prepared queries to eliminate the possibility of code injection into your queries.  So modify your code like so:

```php
$updateOerl = Oerl::findFirst([
    "StudID = :stuid: AND SysNum = :sysnumber:', 
    'bind' => [
        'stuid'     => $resultStud->StudID,
        'sysnumber' => $result->SysOsbj
    ]
]);
```

**Reply by mis-uep** [reply]
Thanks man..
